weblate/formats/tests/test_formats.py

```diff
@@ -848,6 +849,27 @@ class XWikiPropertiesFormatTest(PropertiesFormatTest):
     EXPECTED_FLAGS = ""
     EDIT_TARGET = "[{0}] تىپتىكى خىزمەتنى باشلاش"
 
+    def test_new_language(self):
+        self.maxDiff = None
+        out = os.path.join(self.tempdir, f"test_new_language.{self.EXT}")
+        self.FORMAT.add_language(out, Language.objects.get(code="cs"), self.BASE)
+        template_storage = self.parse_file(self.FILE)
+        new_language = self.FORMAT(out, template_storage, Language.objects.get(code="cs"))
```

Comment:
This is reason of the test crash you pass Langauge object instead of language code to the translate toolkit:

```suggestion
        new_language = self.FORMAT(out, template_storage, Language.objects.get(code="cs").code)
```

This is just to show the problem, I will post improvement of whole code block the in separate comment...

Reply:
thanks!